src/tests/ftest/rebuild/container_create_race.py

```diff
@@ -1,284 +1,166 @@
+#!/usr/bin/python3
 """
   (C) Copyright 2020-2021 Intel Corporation.
 
   SPDX-License-Identifier: BSD-2-Clause-Patent
 """
+
+from ior_test_base import IorTestBase
 from avocado.core.exceptions import TestFail
-from apricot import TestWithServers
-from command_utils_base import CommandFailure
-from job_manager_utils import Mpirun
-from ior_utils import IorCommand
-from test_utils_container import TestContainer
 
 
-class RbldContainerCreate(TestWithServers):
+class RbldContainerCreate(IorTestBase):
+    # pylint: disable=too-many-ancestors
     """Rebuild with container creation test cases.
-
     Test Class Description:
         These rebuild tests verify the ability to create additional containers
         while rebuild is ongoing.
-
     :avocado: recursive
     """
 
-    def add_containers_during_rebuild(self, loop_id, qty, pool1, pool2):
+    def add_containers_during_rebuild(self, qty=10, index=-1):
```

Comment:
Any reason you used index 1? We could have used default index=0 for iterating the container array... Is it not?

Reply:
Thanks for the review and comment.
"index -1" was chosen for the latest container created by default.